auction-app-frontend/src/components/BreadCrumbsBar/BreadCrumbsBar.js

```diff
@@ -29,6 +29,15 @@ const BreadCrumbsBar = ({ title }) => {
                                         {breadcrumb}
                                     </NavLink>
                                     {i + 1 !== data.length && <p> &#8594; </p>}
+                                    {
+                                        match.pathname === '/account' && i+1 === data.length &&
+                                        <>
+                                            <p> &#8594; </p>
+                                            <NavLink to={'/account'}>
+                                                Profile
+                                            </NavLink>
+                                        </>
+                                    }
```

Comment:
Since `/account` refers to Profile tab of the My Account page, and the breadcrumbs bar should look like `My Account -> Profile` as per design, I used this handling to achieve that. This code ensures that  `-> Profile` is added when the location path is `/account`. Without it the breadcrumbs bar would only contain the `My Account` part.